### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by kelvinuche10** [community/community#48674] [GitHub Education]
Hey everyone,
My name is kelvin, I'm new here.
I recently got the GitHub education student pack.
It feels good to be here.

**Comment by jge162** [comment]
welcome.

**Reply by kelvinuche10** [reply]
Hey Jermyiah,
Thanks for the warm welcome.
I'm a beginner, I just got the student pack, and I have been at lost on how to use it to optimize my learning process.
Please, can I use your help.

**Comment by NeemiasBorges** [comment]
Welcome Kelvin!
